src/libcprover-cpp/verification_result.cpp

```diff
@@ -149,3 +150,21 @@ verification_resultt::get_property_status(const std::string &property_id) const
   }
   UNREACHABLE;
 }
+
+// FOTIS' note: Modelled after `result_to_exit_code` in
+// `src/goto-checker/properties.cpp`.
```

Comment:
Should this be refactored to use the same code then?

Reply:
We can't do that because the functions use different types as inputs.

We have two types that are equivalent, but one of them is internal to CBMC (`goto-checker`, really), and thus, we can't expose it to the C++ API (because of the internal headers issue), and the other is the C++ API equivalent, which is present in the headers for the API.

We have in the API (under `verification_result.cpp`) a function which does the translation from `result_internal -> result_external`, but we needed to have a function from `result_external -> BMC_EXIT_CODE`. This is the function that was modelled under the equivalent one from `result_internal -> BMC_EXIT_CODE`.